releasenotes/notes/0.20/Operation-abstract-base-class-99f46180c7ddaa6b.yaml

```diff
@@ -0,0 +1,13 @@
+---
+features:
+  - |
+    Added a new :class:`~qiskit.circuit.Operation` abstract base class for
```

Comment:
I think we should remove this - given that we'll roll-back the usage of `Operation` class for 0.20

Reply:
Ok sure, I'll drop the release note.

Reply:
I removed it here actually: #7840